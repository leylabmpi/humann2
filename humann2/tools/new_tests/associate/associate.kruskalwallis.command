humann2_associate \
		  -i associate.input \
		  -o associate.kruskalwallis.output \
		  -L CATE \
		  -m CATE \
		  -t categorical \
		  -f 0.2 \