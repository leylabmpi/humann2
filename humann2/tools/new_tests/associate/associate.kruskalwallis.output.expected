# FEATURE	EFFECT_SIZE	P-VALUE	Q-VALUE	LEVEL_MEANS
F004	5.87	0.0154	0.06162	0:0; 2:1.054
F010	5.87	0.0154	0.06162	0:0; 2:0.6224
F016	5.87	0.0154	0.06162	0:0; 2:0.6191
F018	5.87	0.0154	0.06162	0:0; 2:0.8907
F020	5.87	0.0154	0.06162	0:0; 2:0.9309
F019	4.688	0.03037	0.1012	0:0.5369; 2:0.2932
F007	3.753	0.05271	0.1506	0:0.7805; 2:0.3547
